xbmc/cores/VideoPlayer/DVDSubtitles/DVDSubtitlesLibass.cpp

```diff
@@ -102,14 +102,16 @@ void CDVDSubtitlesLibass::Configure()
 
   // Load additional fonts into Libass memory
   CFileItemList items;
+  CFileItemList itemsTemp;
   // Get fonts from system directory
   XFILE::CDirectory::GetDirectory(UTILS::FONT::FONTPATH::SYSTEM, items,
                                   UTILS::FONT::SUPPORTED_EXTENSIONS_MASK,
                                   XFILE::DIR_FLAG_NO_FILE_DIRS | XFILE::DIR_FLAG_NO_FILE_INFO);
   // Get temporary fonts
-  XFILE::CDirectory::GetDirectory(UTILS::FONT::FONTPATH::TEMP, items,
+  XFILE::CDirectory::GetDirectory(UTILS::FONT::FONTPATH::TEMP, itemsTemp,
```

Comment:
can you explain why the second vector is required here?

Reply:
when you call GetDirectory appears to delete the previously items added by previous call

Reply:
It might be because `DIR_FLAG_BYPASS_CACHE` is not provided I guess ,and the app just uses the previously cached values for the provided directory